tests/hp/p_adaptivity_flags.cc

```diff
@@ -181,6 +181,28 @@ test()
     validate(tria, dh);
   }
 
+  deallog << "p-adaptivity fixed number" << std::endl;
+  {
+    Triangulation<dim>  tria;
```

Comment:
Just put it into a separate test. this makes it easier to find bugs.

Reply:
I see. While there, I will divide this particular test into multiple ones, each checking one particular component.